src/enums/tokens.ts

```diff
@@ -1,20 +1,20 @@
 export enum Tokens {
```

Comment:
just some reorder for better presentation, stables together, eth related together, etc.